### DISCUSSION TITLE
Help me Redesign my github profile

### DISCUSSION THREAD

**Discussion by NanduBit** [community/community#126269] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Look at my profile and help me Redesign my github profile.
For ans.
Just post it's markdown 
Or Pull request to the profile repo of mine

**Comment by judahpaul16** [comment]
Use a bunch of tables and badges. You can compact a lot of information that way. [Base64Guru](https://base64.guru/converter/encode/image) is your friend. Use my [`README`](https://github.com/judahpaul16) ([raw](https://raw.githubusercontent.com/judahpaul16/judahpaul16/main/README.md)) as a template. I cribbed some designs from drkostas's [`README`](https://github.com/drkostas) ([raw](https://raw.githubusercontent.com/drkostas/drkostas/main/README.md)).

***Replace with your projects and contributions***:

<p align="center">
<a href="https://github.com/NanduWasTaken">
    <img src="https://readme-typing-svg.demolab.com?font=Georgia&size=18&duration=2000&pause=100&multiline=true&width=500&height=80&lines=NanduWasTaken;Computer Scientist+%7C+Philosopher+%7C+Open+Sourcerer;More%20%40%20https://tinyurl.com/NanduWasTaken" alt="Typing SVG" />
</a>
<br/>

<a href="https://tinyurl.com/NanduWasTaken">
    <img src="https://img.shields.io/badge/Website-tinyurl.com/NanduWasTaken-red?style=flat-square">
</a>  
<a href="https://tinyurl.com/NanduWasTaken/resume.pdf">
    <img src="https://img.shields.io/badge/PDF-CV-red?style=flat-square&logo=adobe">
</a>  
<a href="http://www.linkedin.com/in/NanduWasTaken">
    <img src="https://img.shields.io/badge/-Linkedin-blue?style=flat-square&logo=linkedin">
</a>
<a href="#">
    <img src="https://img.shields.io/badge/-Email-red?style=flat-square&logo=gmail&logoColor=white">
</a>
<a href="https://pypi.org/user/NanduWasTaken/">
    <img src="https://img.shields.io/badge/PyPi-NanduWasTaken-blue?style=flat-square&logo=pypi&logoColor=white">
</a>
<a href="https://hub.docker.com/u/NanduWasTaken">
    <img src="https://img.shields.io/badge/DockerHub-NanduWasTaken-blue?style=flat-square&logo=docker&logoColor=white">
</a>

<br/> 


<p align="center">
    <a href="https://github.com/NanduWasTaken">
        <img src="https://github-stats-alpha.vercel.app/api?username=NanduWasTaken&cc=2d333b&tc=37BCF6&ic=fff&bc=0000" alt="NanduWasTaken's GitHub Stats">
    </a>
    <img height="195" src="https://github-readme-stats.vercel.app/api/top-langs/?username=NanduWasTaken&theme=transparent&hide=jupyter%20notebook,cmake,css,c%2b%2b&border_color=2d333b&layout=donut" alt="Top Languages">
</p

</p>

* 📖 I’m currently learning Secret Stuff!

* 📫 How to reach me [^-^](https://tinyurl.com/NanduWasTaken)
  
* <img src='https://github.com/judahpaul16/judahpaul16/raw/main/hi.gif' alt='Hi' width="20"/>&nbsp; I'm Nandu, a developer, self-taught beginner and passionate about IT. By the way, I'm mainly "specialized" in javascript and python.

<table align="center">
<tr><th>Open Source Projects & Contributions</th></tr>
<tr><td>

| Project Name           | Stars | Technologies |
|------------------------|-------|--------------|
| [COVID Tracking](https://github.com/judahpaul16/covid-tracking) | ![Stars](https://img.shields.io/github/stars/judahpaul16/covid-tracking?style=flat-square&labelColor=black) | ![Python](https://img.shields.io/badge/Python-black?style=flat-square&logo=python) ![Tkinter](https://img.shields.io/badge/Tkinter-black?style=flat-square&logo=python) ![Matplotlib](https://img.shields.io/badge/Matplotlib-black?style=flat-square&logo=python) ![Pandas](https://img.shields.io/badge/Pandas-black?style=flat-square&logo=pandas) ![Numpy](https://img.shields.io/badge/Numpy-black?style=flat-square&logo=numpy) ![GNUPlot](https://img.shields.io/badge/GNUPlot-black?style=flat-square&logo=gnu) |
| [Hackermans](https://github.com/judahpaul16/hackermans) | ![Stars](https://img.shields.io/github/stars/judahpaul16/hackermans?style=flat-square&labelColor=black) | ![PhaserJS](https://img.shields.io/badge/PhaserJS-black.svg?style=flat-square&logo=data:image/x-icon;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABmJLR0QAAAAAAAD5Q7t/AAAACXBIWXMAAABIAAAASABGyWs+AAAACXZwQWcAAAAQAAAAEABcxq3DAAABJ0lEQVQ4y6WTP0sDQRDF30hSXoqxuS9gLwoWAQmYwsIIQtCrDdho4LrEgJXNql3Aa43YGVuvSOE16dPlO6Q6OzsZi1z2suxeAubBNLv7fsyfHRIRbCzFLIr5X6SSYpbb622g5gPBVHppSvp2MDShlxdkERSziOfN424nz2QwlPBHjLCAFiCDVKpNbVqoCLLVS1OT2J85a+2/f7h7AACnu/vm6e/cEAbnoFe3cVniUqfbElcfKtWmANBlUAZwkpM4wmg8Mc6OD/dwdHIDIgIAIgDS6bacj1YpiSPUG217jOJ5ophlnb4+nwWAlAAAoQ/UfDwEU6BcxsHbPZI4WplBvdHOS8j+A76vznQJRRqNJ3h6fFn0D1YPisxLRm3WgLWDzmXtAm26zn+ZquF6a4ZkbAAAACJ6VFh0U29mdHdhcmUAAHjac0zJT0pV8MxNTE8NSk1MqQQAL5wF1K4MqU0AAAAASUVORK5CYII=) ![NodeJS](https://img.shields.io/badge/NodeJS-black?style=flat-square&logo=nodedotjs) ![Webpack](https://img.shields.io/badge/Webpack-black?style=flat-square&logo=webpack) ![TypeScript](https://img.shields.io/badge/TypeScript-black?style=flat-square&logo=typescript) |
| [GPT Home](https://github.com/judahpaul16/gpt-home)  <a href="https://hub.docker.com/r/judahpaul/gpt-home"><img src="https://img.shields.io/badge/DH-gray?style=flat-round&logo=docker&logoColor=white"></a> | ![Stars](https://img.shields.io/github/stars/judahpaul16/gpt-home?style=flat-square&labelColor=black) | ![RaspberryPi](https://img.shields.io/badge/Raspberry%20Pi-black?style=flat-square&logo=raspberrypi) ![Python](https://img.shields.io/badge/Python-black?style=flat-square&logo=python) ![FastAPI](https://img.shields.io/badge/FastAPI-black?style=flat-square&logo=fastapi) ![React](https://img.shields.io/badge/React-black?style=flat-square&logo=react) ![NodeJS](https://img.shields.io/badge/NodeJS-black?style=flat-square&logo=nodedotjs) ![TypeScript](https://img.shields.io/badge/TypeScript-black?style=flat-square&logo=typescript) <br> ![OpenAI API](https://img.shields.io/badge/OpenAI%20API-black?style=flat-square&logo=openai) ![Spotify API](https://img.shields.io/badge/Spotify%20API-black?style=flat-square&logo=spotify) ![Philips Hue API](https://img.shields.io/badge/Philips%20Hue%20API-black?style=flat-square&logo=philipshue) ![OpenWeatherMap API](https://img.shields.io/badge/OpenWeatherMap%20API-black.svg?style=flat-square&logo=data:image/x-icon;base64,AAABAAEAEBAAAAEAIABoBAAAFgAAACgAAAAQAAAAIAAAAAEAIAAAAAAAAAQAABILAAASCwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABQcusAT3LrCE1v63FNcOuUTXDrkk1w65BNcOuPTXDrkU1w65JNcOuSTXDrlE1v61pph+0BXX3tAAAAAAAAAAAAUnTrAFJz6wZMb+tYS27rdEtu64RLbuvsS27r/0tu68NLbutzS27rdEtu63VMb+tGnK/1AGGA7QAAAAAAUXLrAFBy6wlMbutyS27rj0tu64pLbuuZS27r70tu6/9LbuvQTG/rdkVp6y0/ZOqGP2TqXUBl6jM/ZOqQQGXqWlFz6wBRc+sGTW/rZUtu6+hLbuv/S27r/0tu6/9Lbuv/S27r7U1v629GausiQWXqbEBl6kpBZuonQWXqcEFm6kZMb+tbTG/rkUxv65hLbuvrS27r/0tu6/9Lbuv/S27r/0tu6/BMb+ucTG/rj01v641Nb+uOTG/rk01v62lSc+sHTXDrTktu6+BLbuv/S27r/0tu6/9Lbuv/S27r/0tu6/9Lbuv/S27r/0tu6/9Lbuv/S27r7ktu64VLbutST3HrBVh47BFMb+vJS27r/0tu6/9Lbuv/S27r/0tu6/9Lbuv/S27r/0tu6/9Lbuv/S27r/0tu6+NQcusiT3HrAAAAAABUdewIS27rtEtu6/9Lbuv/S27r/0tu6/9Lbuv/S27r/0tu6/9Lbuv/S27r/0tu6/9LbuvTUHLrFVBy6wAAAAAARmrqAEtu63dLbuv/S27r/0tu6/9Lbuv/S27r/0tu6/9Lbuv/S27r/0tu6/9Lbuv/S27rm1l57AJSdOwAAAAAAEtu6wBMbusiS27r00tu6/9Lbuv/S27r/0tu6/9Lbuv/S27r/0tu6/9Lbuv/S27r50xv6zpLbusAZ4TuAAAAAABOcesAR2rrAEtu60pLbuvhS27r/0tu6/9Lbuv/S27r/0tu6/9Lbuv/S27r7ktu62VRcuwBTW/rAAAAAAAAAAAAAAAAAE1w6wBWdusAS27rOUtu661LbuvrS27r+0tu6/xLbuvwS27rvExu60xNcOsDTG/rAAAAAAAAAAAAAAAAAAAAAAAAAAAAfJryAEhs6gBPcesJTXDrL01w61BNb+tTTXDrNk9x6w1McOwAUXLrAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA//8AAP//AADAAwAAwAMAAIAAAACAAAAAAAAAAAAAAAAAAwAAAAMAAIADAACABwAAwAcAAMAPAADwPwAA//8AAA==) <br>![GNU Bash](https://img.shields.io/badge/GNU%20Bash-black?style=flat-square&logo=gnubash) ![SystemD](https://img.shields.io/badge/SystemD-black?style=flat-square&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAOxAAADsQBlSsOGwAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAEHSURBVDiNxZOxSsNQFIa/e9OkgVKhxWJNQHIhNNBXEN18CzsVH8FRHBx8A7fikPdwcfIFKk5ONXbpImja6L0OoaVd2sgd/MfL+f7Dx+GKOI7rxadzIzDnwCHVkoFJGy3vymn6nVuBuQSaFWHKWXFc5NoTUdDLgO4f4PW8SwsYoCstYACsC2oAr5MXAFSYINt1/GEfgHw0Rs/mONEe7kmAexrwcfGwMV9bNqkwAcAf9vHOjspHA3r6hT9INraqMFmVbFcwBrP42a2w1FBhQj4agynh/P4ZPZvz/TRdKawrA4go6JmtK3bk/88ogcyCn0gwqUVB6uwftB91gQdCUf1HvgF3jZZ7/QtoQUrOOY/58AAAAABJRU5ErkJggg==) ![NGINX](https://img.shields.io/badge/NGINX-black?style=flat-square&logo=nginx) ![Docker](https://img.shields.io/badge/Docker-black?style=flat-square&logo=docker) |

</td></tr></table>

<table align="center">
<tr><th>Closed Source Projects</th></tr>
<tr>
    
<td>
    
| Project Name       | Technologies |
|--------------------|--------------|
| [FluidVoIP.com](https://fluidvoip.com/)      | ![Python](https://img.shields.io/badge/Python-black?style=flat-square&logo=python) ![Django](https://img.shields.io/badge/Django-black?style=flat-square&logo=django) ![HTMX](https://img.shields.io/badge/HTMX-black?style=flat-square&logo=htmx) ![HyperScript](https://img.shields.io/badge/HyperScript-black?style=flat-square&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACgAAAAYCAYAAACIhL/AAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAVwSURBVFhH7ZdpbBVVFMfn3tnevNf3eJZWSlkSIEJbQIMQNSwmmEKikLAEEWorRNRGY8IHTVBobKW0YDAxEA0hREWhVFYFTSFRwCZVgywu0FqMaEUsW6ELfcvsnnPftMxkWtriFz/4Sybzf/ynM2fuWe7A/c+/hDjnHin78Bvpq1N/rUxqRkgUqJURVd7/fP2Cvx2bsWLT6ayrbeoK3bRFuCb+yNjouyVLcuOOzXj1g7M5qm4tQS1Q8mdzi7599+uTbWb2AXXOPXL+Yusk3TQrQZbylBaFFSlluFANe7Zp2eUgSwWezLUskkw5KUqrGnjL5gpR4kEoNzEr3X+f3rhjgG2d2lzTtAXUiiwcHZYZucwMh1Uf1dO4ahRBAIQQzg7J/L43CnIsx2YYph22LHuB89MUKa3eVDyxX6uH9Brgssoj98aS2kLUlBIuY1Bg38YXZ3ge3hnXsyF101DzhDQHZf5LZrhQDWuyZdujUMMK/6oE+DPM6Ce9BnitLTZB1cwxqAMi35Q1OPQdM1y0x/U5sEIKalmiPw0OB35jhgsIbp5tc3iNDQV/oGxprply+kevAVKOLIXakkHaAVmoeeflmbGUk6L4vR+CHXFjEWrsNFnkqzcsz9PxdxdrqxsV0+LmoYYSaIdrDjFjAPQY4MrNx+/piGuPo8bOjASl/cxwEUua42D1pqAWBNqaGZF86e1M6DOh/oajpoSchSDPMWMA9BjghebW6ZDebNQQYENmVPmRGS4Mw5oH6YugFnlydHPxA1eZ4fDSlp+JZtq4euwZUMcHIMAE6oHgC3DV1lqqadZ8eDibkaGA9PWWV2bdZKbDmqoGEYr/CagtCg/WQwH+M8fqJi1AR9i2PRs19FhSEujhiqLx/e7eLnwBXm6JjYyr+izUAk/1zEFKFTNcNF9P3J/UzEmoYfBeGjJIPsYMF/CCM+EFRqKGGfQtrLKvgfqDL0AYHdN0wxqGOiDx9dmZaY3McFi9/RzpSBhzYPax+SiL9HgkKF5hpguovfmwXHh/E1b5UHlh3oC6twtPgBU7ThBojqehe6GmMb3iwQ3FM1THZrQljAg8/EnUcI0VDgq7SgtyPalb/fG5YYZl56OG9N7giX2QGXeBJ8CLV9uHJlTjYdSQ3hY4fGMBRsskGM6jUUNdXRoxWDnJDBeqbudDetNQQ3q/hzLw7EADwRPglRvxfFU301HD1vbLhFEZ9cxwAf5CWOEgalmgR9YW5nUww6F0Z4MM9bfY+YkvcWD98omeLAyE7gCXVR5WbiW0pc5PLpoW2F/xwnTPjct3NQY03WLDmackJol0DzNcxDRzLLzAQ6ihTtogwC+YcZd0B3izQx3tSm9HNE32pbfpevxRSG8WanhwU3pYOsUMF5BaTG8GaqjRY+uK8q4z4y5hnYiYljUbdoYo6jRFPJ2dEW5ihkPxxjPkRlxfBN3A5iMEWAMB3mKmQ0lVvRBLmF0NZEo8+ZQZLir2NLK/74s1i3NY47EVfG1rrRRP6gtgsBJInRmUhb3rnpvq6UwrQEbC6rHtDzpTj4aEg28W5Hq+bnTdHgfpfRA1RPE73Ms3H2F1i+G0ra+j/JPGGXBOBXitNTkOtrZU3UDHpUcU/76aNKbDCg9BDZt+fSTkTy/48yEA/MDA+9TKErnGDC9j4JrH+jrgDcfixSzA9k71Pt20cFzUweyrGZoZ+gP/3Q0M7+HQnSdA1sHK7Hj72QmeBlq3uxG7F1+gDg/49qsuKxhvMNPLBTjw5e582FwLnG//n2TK8zu7v8NPbSvUHNnN4rdOkvaYJqKGALmasqm+ayr3nqeXbyZZXYcVwax8Zrxv94BPMELp7dq/EyVP5Xg+3/6DcNw/HCg+oSthswAAAAAASUVORK5CYII=) ![jQuery](https://img.shields.io/badge/jQuery-black?style=flat-square&logo=jquery) ![Postgres](https://img.shields.io/badge/Postgres-black?style=flat-square&logo=postgresql) ![SystemD](https://img.shields.io/badge/SystemD-black?style=flat-square&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAOxAAADsQBlSsOGwAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAEHSURBVDiNxZOxSsNQFIa/e9OkgVKhxWJNQHIhNNBXEN18CzsVH8FRHBx8A7fikPdwcfIFKk5ONXbpImja6L0OoaVd2sgd/MfL+f7Dx+GKOI7rxadzIzDnwCHVkoFJGy3vymn6nVuBuQSaFWHKWXFc5NoTUdDLgO4f4PW8SwsYoCstYACsC2oAr5MXAFSYINt1/GEfgHw0Rs/mONEe7kmAexrwcfGwMV9bNqkwAcAf9vHOjspHA3r6hT9INraqMFmVbFcwBrP42a2w1FBhQj4agynh/P4ZPZvz/TRdKawrA4go6JmtK3bk/88ogcyCn0gwqUVB6uwftB91gQdCUf1HvgF3jZZ7/QtoQUrOOY/58AAAAABJRU5ErkJggg==) ![NGINX](https://img.shields.io/badge/NGINX-black?style=flat-square&logo=nginx) <br> ![VoIP.ms API](https://img.shields.io/badge/VoIP.ms%20API-black?style=flat-square&logo=voipdotms) ![Stripe API](https://img.shields.io/badge/Stripe%20API-black?style=flat-square&logo=stripe) ![PayPal API](https://img.shields.io/badge/PayPal%20API-black?style=flat-square&logo=paypal) ![ReCAPTCHA](https://img.shields.io/badge/ReCAPTCHA-black?style=flat-square&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAODSURBVFhHzZdbSFRBGMdnZncVs9SE8CEoqMxkSSij1BR3M7vQU1AGUW/dLFK6vHRFEisoDIq8vPdQSoXQQxJiuism6T5IKKhZSk+Gd4va9uz0n3PGw+ru2T2rC/WD2fm+b3bP/Ge+mTmz5F9DZa1zsGI86Zc1vpoymiRDYZkZGfnNUcEc4Ix7ktLGOt9XOH1aa2SCBAic935kUz9/B3O1FjFmenRUWjrjKC+4hT/y1NiHtJAxIQUIzIoIIWAehVD61Gqz3up6ki5mKCRM1sH4lDnom5DeUrAQzst83j/d2Wf6t8qYzq6Lg2qKQwoouvszlzLWRQjfKEPLIZ1Q7tpe2pcjfbLtbH8uhFULOygFRVVzWVhULpimFmGYFCxmCoPKUfw8lRH+Fulp7qnLLFkwA3vvTyRzQl/BNNV5lKRwxf9G7Tzg+QsE+JX4yhhNe2go2YTPBYPTBRRXTqej8/PSjYRX1phJfg6/E/n8rEWiQxegMOslVBbNM4CSMc75EUqZA566tbrr7PU99fYrGyYyMzjhJxGaFXGzqAIcFdyKURxTI8bMUk6KW2+uetlyfUUnZ3QPYpNaEyGNjVTx1NufccbyIfS7DEdEFWCJm81FlSpsI9B5acuNlb3SJa3XEnswE4ekq+Op3dJLOT8t3YioAvyc7VA9Q/hw6ubE59LRETMhzQUohI2h0tdJOOQaoBFWPn3dWEIV6YRFHDJyq8VpkfCoAijhyapnAKV0UJphCejc9DminoTt7e0NqI4KOwIzjLH9+fn5H6S/bGQKTBHzzgVmBXj9fv+BWHcuMCsgDqNfK+2YEk0KarBWsqRtiNvtXofT0vRzoxGwBsUNEScaGhqCjmwRQ9sppOqTy+WqhwjD21Yg0e6CeYZRmlAGsEW96CwT9mGUwPOktqCg4ALacb0wZqkCTAFhjwsLC8ulG5LFKRBvuIg3WbNg9GUY3APphiRQwIzYaviReMFMaaGYcBUiqqQdhCoAnaqHjMPh6ETeBiBkH8L6qzYGlLW1teHCE4wqwGazXQ48ZCDkI/K3G2afFlkWX/CsPKyFkO+TsFulubk5MSEh4TZMcVuyqUHzeDGztRaL5U5eXp7h/wtTe7Wjo2O9oijlGMlxPDRNho34hu80+Xy+h06n86uMGWJKwDwQwCBmJ8Rko5MMhFJQRBon0TaEdeRGKj1oC7v3/yMI+QvvnDqvUV+oWgAAAABJRU5ErkJggg==) ![WeasyPrint](https://img.shields.io/badge/WeasyPrint-black?style=flat-square&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEcAAABHCAYAAABVsFofAAAACXBIWXMAAA7DAAAOwwHHb6hkAAAAGXRFWHRTb2Z0d2FyZQB3d3cuaW5rc2NhcGUub3Jnm+48GgAAA2xJREFUeJztnE1rE1EUht9zU3+BILUfM9MW3LsRXQhaFV278QcIRZPUCtqqWGtFF36ASJtpq+APcO1CRFERRMStHxspTdJOTQV/gE3ucRHth86ctjDtvXTus5z3Lt48k1zmHpIQM8M0PfNj3qJWTwi0D4Ay3YeBCjT3tZguAgB1VsME2m+6x18I8FjRI+N3CQAI1Gm6w78Q4Fkhx1acHAEnR8DJEXByBJwcASdHwMkRcHIEnBwBJ0fAyRFwcgScHAEnR8DJEXByBJwcgaUZcjBX6tXAoZYdjXB610DNZClbUADgz5YGGPSCQNcaiy0fglqpy3QxG1DB3MQtED3A8kfM5zq97Jkf80wWswHF4LMx17vrOvc664IUg8OErLveyL3png/9LW1kEarSXhwB4U5sSuiqa7zN6h6kAKDcVricJIgAT9fpjV+Z7N7aauZZes6ptBWugDAWt4gAj3P6ddYELclhgCtthfNO0DKrnpBXCBqPW5w1Qf8dH/4IGlhLkFcd79n8emaJPVv9FcTMpbicAI+U2vaCEg+eDHC1o3iOwBMJSzqh1Kuu71PB5lQzj3gqZ4DL7cUiAZNxOQEeNxrPO+bCnZtTzyxrjiyaggoFgKYS8j0K/Cj9auZZ1zyHAa605/MAHsblBDoZRON7U21mAesedjUFFc4m7UGs1cH0atnBhiaBDLAG/YwNaftNFTf0gvzZ8AYBw3EZa3qXTiV7WLccf7Y0CsJIQvy00pn/mFIna1iXnKYYup4QT//K6dMpdrKGNeX40cR1QUykmY7Pt/b/SLmXFYjfYPdnw0sgjCbENeT0sWpr/7f0a9lB4jvHj0pDINxOiGvI6d5ya/+XTeplBbFy/Kg0BKb40WlGxAAxcoIoHHRimqySE0ThIDPuJqzNlBhghZwgKl0UxCyggSNZEgP8kdMUQ/cS1iyggd6yV/i8hb2soCWYm7jASBRTI0WHZ9rzX7e0lSUoBl9NyGrI6d6Z3dkUAwCKmR7HXM/c5huHqnbkh8B8Y8W1Bda5o1kXAwCKAS53FEcBnCLgPhrqQKXzzCfTxWyAbPjpdBCFz5hxwnSPf9l207s0cXIEnBwBJ0fAyRFwcgScHAEnR8DJEXByBJwcASdHwMkRsEIOg6umO8RQtkJOHeomgPcAtOkuQPPf3lhR32/5giY4HMrhHgAAAABJRU5ErkJggg==) ![Jinja](https://img.shields.io/badge/Jinja-black?style=flat-square&logo=jinja) ![Docker](https://img.shields.io/badge/Docker-black?style=flat-square&logo=docker) |
| [Atlexecpm.com](https://atlexecpm.com/) | ![React](https://img.shields.io/badge/React-black?style=flat-square&logo=react) ![Express](https://img.shields.io/badge/Express-black?style=flat-square&logo=express) ![NodeJS](https://img.shields.io/badge/NodeJS-black?style=flat-square&logo=nodedotjs) ![TypeScript](https://img.shields.io/badge/TypeScript-black?style=flat-square&logo=typescript) ![ReCAPTCHA](https://img.shields.io/badge/ReCAPTCHA-black?style=flat-square&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAODSURBVFhHzZdbSFRBGMdnZncVs9SE8CEoqMxkSSij1BR3M7vQU1AGUW/dLFK6vHRFEisoDIq8vPdQSoXQQxJiuism6T5IKKhZSk+Gd4va9uz0n3PGw+ru2T2rC/WD2fm+b3bP/Ge+mTmz5F9DZa1zsGI86Zc1vpoymiRDYZkZGfnNUcEc4Ix7ktLGOt9XOH1aa2SCBAic935kUz9/B3O1FjFmenRUWjrjKC+4hT/y1NiHtJAxIQUIzIoIIWAehVD61Gqz3up6ki5mKCRM1sH4lDnom5DeUrAQzst83j/d2Wf6t8qYzq6Lg2qKQwoouvszlzLWRQjfKEPLIZ1Q7tpe2pcjfbLtbH8uhFULOygFRVVzWVhULpimFmGYFCxmCoPKUfw8lRH+Fulp7qnLLFkwA3vvTyRzQl/BNNV5lKRwxf9G7Tzg+QsE+JX4yhhNe2go2YTPBYPTBRRXTqej8/PSjYRX1phJfg6/E/n8rEWiQxegMOslVBbNM4CSMc75EUqZA566tbrr7PU99fYrGyYyMzjhJxGaFXGzqAIcFdyKURxTI8bMUk6KW2+uetlyfUUnZ3QPYpNaEyGNjVTx1NufccbyIfS7DEdEFWCJm81FlSpsI9B5acuNlb3SJa3XEnswE4ekq+Op3dJLOT8t3YioAvyc7VA9Q/hw6ubE59LRETMhzQUohI2h0tdJOOQaoBFWPn3dWEIV6YRFHDJyq8VpkfCoAijhyapnAKV0UJphCejc9DminoTt7e0NqI4KOwIzjLH9+fn5H6S/bGQKTBHzzgVmBXj9fv+BWHcuMCsgDqNfK+2YEk0KarBWsqRtiNvtXofT0vRzoxGwBsUNEScaGhqCjmwRQ9sppOqTy+WqhwjD21Yg0e6CeYZRmlAGsEW96CwT9mGUwPOktqCg4ALacb0wZqkCTAFhjwsLC8ulG5LFKRBvuIg3WbNg9GUY3APphiRQwIzYaviReMFMaaGYcBUiqqQdhCoAnaqHjMPh6ETeBiBkH8L6qzYGlLW1teHCE4wqwGazXQ48ZCDkI/K3G2afFlkWX/CsPKyFkO+TsFulubk5MSEh4TZMcVuyqUHzeDGztRaL5U5eXp7h/wtTe7Wjo2O9oijlGMlxPDRNho34hu80+Xy+h06n86uMGWJKwDwQwCBmJ8Rko5MMhFJQRBon0TaEdeRGKj1oC7v3/yMI+QvvnDqvUV+oWgAAAABJRU5ErkJggg==) |
| [JudahPaul.com](https://judahpaul.com/) | ![PhaserJS](https://img.shields.io/badge/PhaserJS-black.svg?style=flat-square&logo=data:image/x-icon;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABmJLR0QAAAAAAAD5Q7t/AAAACXBIWXMAAABIAAAASABGyWs+AAAACXZwQWcAAAAQAAAAEABcxq3DAAABJ0lEQVQ4y6WTP0sDQRDF30hSXoqxuS9gLwoWAQmYwsIIQtCrDdho4LrEgJXNql3Aa43YGVuvSOE16dPlO6Q6OzsZi1z2suxeAubBNLv7fsyfHRIRbCzFLIr5X6SSYpbb622g5gPBVHppSvp2MDShlxdkERSziOfN424nz2QwlPBHjLCAFiCDVKpNbVqoCLLVS1OT2J85a+2/f7h7AACnu/vm6e/cEAbnoFe3cVniUqfbElcfKtWmANBlUAZwkpM4wmg8Mc6OD/dwdHIDIgIAIgDS6bacj1YpiSPUG217jOJ5ophlnb4+nwWAlAAAoQ/UfDwEU6BcxsHbPZI4WplBvdHOS8j+A76vznQJRRqNJ3h6fFn0D1YPisxLRm3WgLWDzmXtAm26zn+ZquF6a4ZkbAAAACJ6VFh0U29mdHdhcmUAAHjac0zJT0pV8MxNTE8NSk1MqQQAL5wF1K4MqU0AAAAASUVORK5CYII=) ![NodeJS](https://img.shields.io/badge/NodeJS-black?style=flat-square&logo=nodedotjs) ![Webpack](https://img.shields.io/badge/Webpack-black?style=flat-square&logo=webpack) ![TypeScript](https://img.shields.io/badge/TypeScript-black?style=flat-square&logo=typescript) ![Python](https://img.shields.io/badge/Python-black?style=flat-square&logo=python) ![Gymnasium](https://img.shields.io/badge/Gymnasium-black?style=flat-square&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAACS0lEQVQ4jW2TT0iUYRDGf/N9arQmxEIHJesQCIWkBQbVIaFDWZHkpUNoRrSBsUGXIlh953UXk5DWyiBQ+0N/LoEHK5Aw6hYRdIggiLp4CMqLh/YQm9908N1wqbm8M8M8w8wzzytmxmoTEVS1G9gAfAzpVmAxnU7PZbPZX1X1qxsMDw93m9kE8BUQoA0w4EN4twDnnHOzFUxUcbz3O83spqo2O+c6ReQe8ANYBKadc52q2gzc8t63/9MAyAAjAMVica2ZjcRx3BXH8WHgasghIqPA6aoVvPctwAzwDZiPomghSZLjzrljYbpZ4KGIbDazg4GfHufcF1HV7cBz4DXQCewF3gHTQGXXHqAP2AW8AV4C+6IoOhIBV4BLqtoLvAfuAB1Af2C/FTgRwNPAW1U9KSK5JEkKEbBbVR+bWeV8W4FJYFxVp1R1CrgO3AdayuVy1swslUrNAB0RUAIYGxurV9VPwADQFUauWC+wHzhfW1v71HtfVyqVGoCyqOo4gIikzKzOOdcfiMsB50REzGzCOVcIIquQ/RswUdVtwAOgSUTUzJ4A64AbwGgongTOAD+jKOpPkuQC8B3oEzOjUCi0LC8vvxKRRTNrAJZYUeMLIA3sAZqB9UDJzBpqamoO5HK5z3+l7L1/JCLzQ0NDd0O8CcgHDgadcwshPwC0OefOQrUSi2Y26L1vDfFGoD74TQD5fH4HcDmO42sVUNVn8t4fAm6zIqAF4BSwhhWJtwNHgYxzbu6/DcI1UNWLYd9nQBKAS42NjcVMJlNeXf8H8pTwOh5DSEEAAAAASUVORK5CYII=) |

</td>

</tr>
</table>

**Comment by ebndev** [comment]
Hey @NanduWasTaken, thanks for being a part of the GitHub Community!

Feel free to get inspiration from the [Profile Showcase](https://github.com/orgs/community/discussions/121833) thread. If you have questions on how they implemented something, try commenting under their post 😄

**Comment by CloudberryPiTechnology** [comment]
If you try replacing black with another color and make the readmes more interesting it would help and also make sure it follows the community requirements tab which is in the insights are of the repository
